clients/drcachesim/CMakeLists.txt

```diff
@@ -512,6 +512,9 @@ macro(add_drmemtrace name type)
   if (liblz4)
     target_link_libraries(${name} lz4)
   endif ()
+  if (RISCV64)
+    target_link_libraries(${name} atomic)
+  endif ()
```

Comment:
@derekbruening I did some debugging as suggested and found that the missing symbol is `__atomic_is_lock_free`. This change makes the `reuse_distance` pass, but I'm not sure if it's a proper way to do it.

Reply:
Yeah, it is weird. After searching, I found [this page](https://blog.jiejiss.com/A-RISC-V-gcc-pitfall-revealed-by-a-glibc-update/) states that: *gcc does not support inlining subword atomics, and this is the reason why -latomic must be presented when invoking gcc.* Maybe that's the reason for this? not sure.